Makefile

```diff
@@ -55,25 +55,20 @@ generate: controller-gen
 # Build the docker image
 #docker-build: test
 docker-build:
-	docker build . -t ${IMG}
+	podman build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
 	sed -i'' -e 's@image: .*@image: '"${IMG}"'@' ./config/default/manager_image_patch.yaml
 
 # Push the docker image
```

Comment:
worth renaming all docker (comments, targets) to podman?

Reply:
Personally, I don't find this target useful anyway.  I just build/push the image with one command.

Reply:
> Personally, I don't find this target useful anyway. I just build/push the image with one command.

+1
btw, where can I find that manager_image_patch.yaml?

@bkhizgiy how about dropping all the docker to podman changes from this PR? we were distracted and this PR had value also without those changes